src/oge/download_data.py

```diff
@@ -292,14 +304,14 @@ def download_epa_psdc(psdc_url: str):
 
 
 def download_raw_eia923(year: int):
-    """
-    Downloads raw EIA-923 data (zip files), and unzips them to the downloads folder.
+    """Downloads raw EIA-923 data (zip files), and unzips them to the downloads folder.
 
-    Inputs:
-        `year`: A four-digit year.
+    Args:
+        year (int): a four-digit year.
     """
     if year < 2008:
-        raise NotImplementedError(f"EIA-923 data is unavailable for '{year}'.")
+        logger.warning(f"EIA-923 data is unavailable for {year}.")
+        return
```

Comment:
Do we want to go ahead and `download_raw_eia_906_920()` here?
Maybe we still want a warning message that says something like "EIA-923 data is not available before 2008. Downloading EIA-906/920 files instead."

Reply:
Sure. But we are not going to use it at the moment

Reply:
That's fine - I'd rather have it downloaded and ready to use in the next PR

Reply:
Done. See below:
```
2024-05-17 13:54:00 [WARNING] oge.oge.download_data:313 EIA-923 data is not available before 2008. Downloading EIA-906/920 files instead
2024-05-17 13:54:00 [INFO] oge.oge.download_data:55 f906920_2005 already downloaded, skipping.
```